api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -270,6 +270,8 @@ public boolean isProductionEnv() {
     public boolean enableResearchPurposePrompt;
     // If true, reporting cron job will write data to configured BigQuery reporting dataset.
     public boolean enableReportingUploadCron;
+    // If true, user account setup requires link account via RAS account instead of eRA Commons.
```

Comment:
nit: I would say that this requires linking eRA commons via RAS instead of Shibboleth.